docs/usage/configuration/config_documentation.md

```diff
@@ -3697,7 +3697,7 @@ As a result, the worker configuration is divided into two parts.
 
 1. The first part (in this section of the manual) defines which shardable tasks
    are delegated to privileged workers. This allows unprivileged workers to make
-   request a privileged worker to act on their behalf.
+   requests to a privileged worker to act on their behalf.
```

Comment:
This is the only intended change to this file in this commit. I believe the rest is just whitespace adjustments my file editor happily did without my knowledge 🙄